src/components/variable.jsx

```diff
@@ -4,6 +4,8 @@ export const colorHackRedMed = "#f56f65"
 export const colorHackRedLight = "#f89790"
 export const colorHackYellow = "#ffd081"
 export const colorHackTeal = "#48DEE2"
+export const purple = '#5C63AB'
+export const purpleLight = '#5C63AB19'
```

Comment:
I'm not sure if this hex code is accurate?

Reply:
yes, apparently the last two are alpha (transparency)